src/lib.rs

```diff
@@ -243,6 +243,12 @@ where
         Ok(())
     }
 
+    /// Set accelerometer reading scale
+    pub fn a_scale(&mut self, scale: FSScale) -> Result<(), E> {
+        self.modify(Register::ACCEL_CONFIG, |r| (r & !0b11000) | scale as u8)?;
```

Comment:
Doesn't `scale as u8` need to be left shifted before OR-ing it with the masked contents?

Reply:
Yeah whoops